src/components/transactions/Switch/SwitchModal.tsx

```diff
@@ -45,8 +45,12 @@ const SwitchModalContentWrapper = ({
 }: SwitchModalContentWrapperProps) => {
   const filteredTokens = useMemo(() => getFilteredTokens(chainId), [chainId]);
 
+  console.log('filteredTokens', filteredTokens);
+
   const { data: baseTokenList } = useTokensBalance(filteredTokens, chainId, user);
 
+  console.log('baseTokenList', baseTokenList);
```

Comment:
remove comment

Reply:
thanks upated in 7ac479d79b025832a32c68c46ad58831e641a652